### DISCUSSION TITLE
Possible to display an image in the error report (browser mode)

### DISCUSSION THREAD

**Discussion by jnsmalm** [vitest-dev/vitest#4156] [Q&A]
Right now, my error looks like this:

<img width="778" alt="image" src="https://github.com/vitest-dev/vitest/assets/10713751/0d2726e7-b36c-404f-9174-c603d80e59ed">

Instead, I would like to display the actual images from those urls. How can I achieve that?

**Comment by Andrew-web-coder** [comment]
See https://github.com/vitest-dev/vitest/issues/4598
